integration-tests/cucumber-tests-platform-smartmetering/src/test/java/org/opensmartgridplatform/cucumber/platform/smartmetering/config/SecretManagementPersistenceConfig.java

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright 2020 Smart Society Services B.V.
+ * Copyright 2020 Alliander.V.
```

Comment:
"Alliander N.V." does not look quite right in this copyright statement ("Alliander.V.").